src/llnode.cc

```diff
@@ -130,7 +130,7 @@ bool BacktraceCmd::DoExecute(SBDebugger d, char** cmd,
 
 bool PrintCmd::DoExecute(SBDebugger d, char** cmd,
                          SBCommandReturnObject& result) {
-  if (*cmd == NULL) {
+  if (cmd == NULL || *cmd == NULL) {
```

Comment:
Can you use nullptr here?  Seems you stumbled on one of the three places that still uses NULL (vs. ~50 instances of nullptr.)

Are other commands affected too?  A quick grep suggests some, like `list` and `findrefs`, don't check `cmd == nullptr` either.